src/VirtoCommerce.Platform.Web/wwwroot/js/common/directives/fallbackSrc.js

```diff
@@ -9,15 +9,23 @@ angular.module('platformWebApp')
                 // Update the service with the correct missing src if present, otherwise use the default image
                 var newSrc = attr.fallbackSrc ? imageService.setMissing(attr.fallbackSrc) : imageService.getMissing();
 
+                var errorCount = 0;
                 // Listen for errors on the element and if there are any replace the source with the fallback source
                 var errorHandler = function() {
 
-                    // fallbackSrc may have changed since the link function ran, so try to grab it again.
-                    var newSrc = attr.fallbackSrc ? imageService.setMissing(attr.fallbackSrc) : imageService.getMissing();
+                    // To prevent recursive fails on loading missed (404) fallbackSrc, try to do it only once
+                    if (errorCount === 0) {
+                        // fallbackSrc may have changed since the link function ran, so try to grab it again.
+                        var newSrc = attr.fallbackSrc
+                            ? imageService.setMissing(attr.fallbackSrc)
+                            : imageService.getMissing();
 
-                    if (element[0].src !== newSrc) {
-                        element[0].src = newSrc;
+                        if (element[0].src !== newSrc) {
+                            element[0].src = newSrc;
+                        }
                     }
+
+                    errorCount++;
```

Comment:
In case of such logic, maybe you could remove event handler after the first handling?